# [Think Stats Chapter 5 Exercise 1](http://greenteapress.com/thinkstats2/html/thinkstats2006.html#toc50) (blue men)

**Exercise 1**   In the BRFSS (see Section 5.4), the distribution of heights is roughly normal with parameters µ = 178 cm and σ = 7.7 cm for men, and µ = 163 cm and σ = 7.3 cm for women.
In order to join Blue Man Group, you have to be male between 5’10” and 6’1” (see http://bluemancasting.com). What percentage of the U.S. male population is in this range? Hint: use scipy.stats.norm.cdf.

In [5]:
import scipy.stats

# 5'10" = 117.8cm
# 6'1"  = 185.42cm
scipy.stats.norm.cdf(185.42,loc=178,scale=7.7)-scipy.stats.norm.cdf(177.8,loc=178,scale=7.7)


0.34274683763147368

The difference between the two CDFs at the respective points is the percentage of the male population that fits that range. So 34.27% of the male population falls within that height range.

Unfortunately for Tobias Funke, actor David Cross is only 5'9".